modules/mono/editor_templates/CharacterBody2D/basic_movement.cs

```diff
@@ -0,0 +1,41 @@
+// meta-description: Classic movement for gravity games (platformer, ...)
+// meta-default: true
+
+using _BINDINGS_NAMESPACE_;
+using System;
+
+public partial class _CLASS_ : _BASE_
+{
+    public const int Speed = 1000;
+    public const int JumpForce = -1000;
+
+    // Get the gravity from the project settings to be synced with RigidDynamicBody nodes.
+    public int gravity = (int)ProjectSettings.GetSetting("physics/2d/default_gravity");
```

Comment:
This cast will throw an exception since it actually returns a `float` but the `GetSetting` returns an `object` so you can't cast from `object` to `int`, you have to cast to `float` first.

This would work:
```suggestion
    public int gravity = (int)(float)ProjectSettings.GetSetting("physics/2d/default_gravity");
```

Reply:
Thanks, I've pushed the modification (but neither fabriceci nor me can currently test that C# templates are running well for a reason unrelated to this PR).

Reply:
Nice spot, I tried with 3.X as 4.0 not works with mono for me, so I didn’t see the setting has changed from int to float!